linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml

```diff
@@ -26,7 +26,7 @@ references:
     cui: 3.1.12
     disa: "366"
     hipaa: 164.308(a)(4)(i),164.308(b)(1),164.308(b)(3),164.310(b),164.312(e)(1),164.312(e)(2)(ii)
-    nist: CM-6(b)
+    nist: CM-7(a),CM-7(b),CM-6(a)
```

Comment:
@shawndwells should `AC-17(a)` be added

Reply:
Yes, ``AC-17(a),CM-7(a),CM-7(b),CM-6(a)``